include/triton/Dialect/TritonGPU/IR/TritonGPUAttrDefs.td

```diff
@@ -324,5 +324,20 @@ def SliceEncodingAttr : DistributedEncoding<"SliceEncoding"> {
   let extraClassDeclaration = extraBaseClassDeclaration;
 }
 
+def DotOperandEncodingAttr : DistributedEncoding<"DotOperandEncoding"> {
+  let mnemonic = "dot_op";
+
+  let description = [{}];
+
+  let parameters = (
+    ins
+    "unsigned":$opNum,
```

Comment:
`opNum` might be confused with `operand number`, as in the number of operands that this layout has. A better name would probably be `opIdx`.

Reply:
+1